docs/README.md

```diff
@@ -79,15 +79,17 @@ The Benefits application doesn't collect or store any user data directly, and we
 
 Sensitive user information exists in the following places:
 
-- To enroll in a senior discount, users need to [provide personal information to Login.gov](https://benefits.calitp.org/help#login-gov-verify).
-- Users need to [provide their credit or debit card information to our transit processor (Littlepay)](https://benefits.calitp.org/help#littlepay) to enroll in a discount.
+- To enroll for reduced fares as an older adult, U.S. Veteran, or Calfresh cardholder, riders need to [provide personal information to Login.gov](https://benefits.calitp.org/help#login-gov-verify).
+- To enroll for reduced fares as a Medicare cardholder, riders need to [provide personal information to Medicare.gov](https://www.medicare.gov/basics/get-started-with-medicare).
+- Eligible riders need to provide their contactless credit or debit card information to one of our transit processors, [Littlepay](https://benefits.calitp.org/help#littlepay) or [Switchio](https://benefits.calitp.org/help#littlepay), to register their card for reduced fares.
```

Comment:
```suggestion
- Eligible riders need to provide their contactless credit or debit card information to one of our transit processors, [Littlepay](https://benefits.calitp.org/help#littlepay) or [Switchio](https://benefits.calitp.org/help#switchio), to register their card for reduced fares.
```

i think creating a generic https://benefits.calitp.org/help#transit_processors page would be a great future-proof option too.